src/App.js

```diff
@@ -1,59 +1,67 @@
-import React from 'react';
-import { Routes, Route, NavLink } from 'react-router-dom';
+import { Routes, Route, useNavigate } from 'react-router-dom';
 import './App.css';
-import Firebase from './Pages/firebase';
+import { getToken } from '@the-collab-lab/shopping-list-utils';
+import { useState, useEffect } from 'react';
+import AddItem from './components/AddItem/AddItem';
+import NavLinks from './components/Navigation/NavLinks';
+import ItemList from './pages/ItemList';
 
-function List() {
-  return (
-    <>
-      <h1>List of Items</h1>
-    </>
-  );
-}
+function App() {
+  const [token, setToken] = useState(null);
+  const navigate = useNavigate();
 
-function AddAnItem() {
-  return (
-    <>
-      <h1>Add an Item</h1>
-    </>
-  );
-}
+  useEffect(() => {
+    const json = localStorage.getItem('dummy');
+    const loadedToken = JSON.parse(json);
+    if (loadedToken) {
+      navigate('/item-list');
+      setToken(loadedToken);
+    }
+  }, []);
 
-function NavLinks() {
-  return (
-    <nav style={{ position: 'fixed', bottom: 0 }}>
-      <NavLink
-        to="/"
-        style={({ isActive }) =>
-          isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
-        }
-      >
-        List of Items
-      </NavLink>
-      <NavLink
-        to="/add-an-item"
-        style={({ isActive }) =>
-          isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
-        }
-      >
-        Add an Item
-      </NavLink>
-    </nav>
-  );
-}
+  useEffect(() => {
+    const json = JSON.stringify(token);
```

Comment:
It looks like we may be setting the token twice. In the `onClick` handler, we're calling `localStorage.setItem` and then updating the state with this token that was just set. 
That, in turn, causes this `useEffect` to be triggered, which sets the token again.

Instead, I think that we may just want to get rid of this useEffect.